10up/simple-podcasting
# Ensure that the selected podcast term doesn't get removed from the podcast block settings.

### Description of the Change
PR fixes the issues reported in https://github.com/10up/simple-podcasting/issues/313#issuecomment-2364568916
- Display podcast term selection only at the post level now (removed showing twice, post level and block level)
- Ensure that the selected podcast term doesn't get removed from the podcast block settings

### How to test the Change
1. Create a post.
1. Add a podcast block.
1. Choose a podcast term from the post sidebar panel and make sure selected term get reflected in editor block preview.
1. Save the post and refresh the page. Ensure that selected podcast terms remains unchanged and same podcast term display in frontend.


### Changelog Entry
> Fixed - Ensure that the selected podcast term doesn't get removed from the podcast block settings.
> Removed - The postcast terms selector from the podcast block settings.

### Credits
Props @dkotter  @iamdharmesh

### Checklist:
